src/containerapp/HISTORY.rst

```diff
@@ -7,6 +7,12 @@ Upcoming
 * 'az containerapp env update': Fix bugs in update environment.
 * Fix YAML create with user-assigned identity
 * Fix polling logic for long running operations.
+* 'az containerapp env create': add support for workload profiles
+* 'az containerapp env update': add support for workload profiles
+* 'az containerapp create': add support for workload profiles
+* 'az containerapp update': add support for workload profiles
+* Add 'az containerapp env workload-profile' commands
```

Comment:
It is suggested to point out what the function of the commands is. 
e.g., Add 'az containerapp env workload-profile' commands to support...